addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -503,12 +519,16 @@ var SnippetEditor = Widget.extend({
         this.trigger_up('activate_insertion_zones', {
             $selectorSiblings: $selectorSiblings,
             $selectorChildren: $selectorChildren,
+            forbidSanitize: self._isForbidSanitize(self.$target),
```

Comment:
Use `this` instead of `self` for your new lines, will be easier to forward-port... and just better :)